src/sfloader/fluid_defsfont.c

```diff
@@ -2030,24 +2034,40 @@ fluid_sample_import_sfont(fluid_sample_t* sample, SFSample* sfsample, fluid_defs
    equivalent to the matching ID list in memory regardless of LE/BE machine
 */
 
+#if FLUID_IS_BIG_ENDIAN
+#define UINT32_FROM_LE(x) \
+  ((((x) & 0xff000000) >> 24) | (((x) & 0x00ff0000) >>  8) | \
+  (((x) & 0x0000ff00) <<  8) | (((x) & 0x000000ff) << 24))
+#define INT32_FROM_LE(x) \
+  ((((x) & 0xff000000) >> 24) | (((x) & 0x00ff0000) >>  8) | \
+  (((x) & 0x0000ff00) <<  8) | (((x) & 0x000000ff) << 24))
+#define INT16_FROM_LE(x) \
+  ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8))
+#else
+#define UINT32_FROM_LE(x) x
+#define INT32_FROM_LE(x) x
+#define INT16_FROM_LE(x) x
+#endif
+
+
 #define READCHUNK(var,fd)	do {		\
 	if (!safe_fread(var, 8, fd))			\
 		return(FAIL);				\
-	((SFChunk *)(var))->size = GUINT32_FROM_LE(((SFChunk *)(var))->size);  \
+	((SFChunk *)(var))->size = UINT32_FROM_LE(((SFChunk *)(var))->size);  \
```

Comment:
I'm not feeling comfortable to start duplicating functionality that is provided by glib.

Reply:
the goal is to try to allow one to get rid of glib.
I can add a define in fluid_sys.h for that then ?

Reply:
Not sure. @mawe42 What do you think of duplicating that byteswap logic?

Reply:
Couldn't @loki666 just define GUINT32_FROM_LE in his own glib-free fluid_sys.h?

Reply:
yes but fluid_defsfont.h still include glib.h

Reply:
Ok... but wouldn't you need to remove that as well, to get rid of the glib dependency?

Reply:
well... I d'like to avoid to replace the fluid_defsfont.c/h just because of the deps on simple GLIB marcos for this... but if you don't want to change that I'll do it in my branch.

I wanted to have a minimum changes in my branch so I can merge from your repository without having too much conflicts

Reply:
Then please `#define FLUID_LE32TOH` in `fluidsynth_priv.h` which replace `G*_FROM_LE`.

Reply:
Ok I'll do that in another PR

Reply:
Nevermind. I'll draft an example.